var/spack/repos/builtin/packages/py-fortls/package.py

```diff
@@ -16,11 +16,16 @@ class PyFortls(PythonPackage):
 
     license("MIT")
 
+    version(
+        "3.1.0", sha256="e38f9f6af548f78151d54bdbb9884166f8d717f8e147ab1e2dbf06b985df2c6d"
+    )  # FIXME
     version("2.13.0", sha256="23c5013e8dd8e1d65bf07be610d0827bc48aa7331a7a7ce13612d4c646d0db31")
 
     depends_on("py-setuptools@45:", type="build")
+    depends_on("py-setuptools@61:", when="@3:", type="build")
     depends_on("py-packaging", type=("build", "run"))
     depends_on("py-setuptools-scm@6.2:+toml", type="build")
+    depends_on("py-setuptools-scm@7:+toml", when="@3:", type="build")
     depends_on("py-setuptools-scm-git-archive", type="build")
 
     depends_on("py-json5", type=("build", "run"))
```

Comment:
If you want, you can move py-packaging down below py-json5, since pyproject.toml has now
```
dependencies = [
    "json5",
    "packaging",
    "importlib-metadata; python_version < '3.8'",
    "typing-extensions; python_version < '3.8'",
]
```